.github/workflows/iam-validate.yml

```diff
@@ -0,0 +1,50 @@
+# Copyright 2023 The Authors (see AUTHORS file)
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#      http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: 'aod-iam-validate'
+
+# Support pull request triggering event.
+on:
+  workflow_call:
+    inputs:
+      aod_cli_version:
+        description: 'The version of AOD CLI.'
+        type: 'string'
+        default: 'latest'
+        required: false
+      go_version:
+        description: 'The version of Golang.'
+        type: 'string'
+        default: '1.20'
+        required: false
+
+jobs:
+  validate:
+    runs-on: ubuntu-latest
+    name: 'Validate IAM Request'
+    steps:
+      - name: 'Checkout Triggering Branch'
+        uses: 'actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab' # ratchet:actions/checkout@v3
+        with:
+          ref: ${{ github.event.pull_request.head.ref }}
```

Comment:
Please quote all YAML strings

Reply:
Ack!

Reply:
I also keep forgetting. https://github.com/abcxyz/readability/wiki/YAML-Best-Practices

Reply:
We should just write a linter... I wonder if I can nerd snipe @drevell into doing it since he just played around with the YAML ast parser for abc?

Reply:
No, we'd put this in abcxyz/pkg as part of our linters (kinda like we did with Terraform).

Reply:
I could conceivably be sniped into doing that after I ship v1 of the abc CLI 😄

Reply:
LOL reminder created https://github.com/abcxyz/pkg/issues/130